src/yunohost/log.py

```diff
@@ -252,8 +252,7 @@ def func_wrapper(*args, **kwargs):
                 result = func(*args, **kwargs)
             except Exception as e:
                 uo.error(e)
-                t, v, tb = exc_info()
-                raise t, v, tb
+                raise
```

Comment:
Have you check it make the same thing ? Because I think I added that due to the miss of the stacktrace.